static/shared/js/typeahead.js

```diff
@@ -98,13 +98,26 @@ export function clean_query_lowercase(query) {
     return query;
 }
 
+export const is_unicode_emoji = (emoji) =>
+    emoji.reaction_type === "unicode_emoji" && Boolean(emoji.emoji_code);
```

Comment:
Why is the `Boolean` part required in this check?

Reply:
Thank you for bringing this up, I think it's not needed indeed.

I got a little carried away and obsessed with converting the value into an actual boolean value. But in hindsight, I see it's not needed.

I have removed it now 👍